packages/plugins/docs-ui/src/lib/components/preview/docs-preview-modal.component.html

```
@@ -24,7 +24,7 @@
 					<nb-icon icon="arrow-ios-forward-outline"></nb-icon>
 				</button>
 			</ng-container>
-			<button nbButton ghost size="small" (click)="download()">
+			<button nbButton ghost size="small" [disabled]="downloading" (click)="download()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)